engine/src/card.cpp

```diff
@@ -1088,30 +1088,43 @@ void MCCard::timer(MCNameRef mptr, MCParameter *params)
 		MCObject::timer(mptr, params);
 }
 
+bool MCCard::isdeletable()
+{
+    bool t_deletable = true;
+    
+    if (t_deletable)
+        t_deletable = MCObject::isdeletable();
+    
+    if (t_deletable)
+        t_deletable = !getflag(F_C_CANT_DELETE);
```

Comment:
Think about this probably deleting a stack should be allowed if the cantDelete of a card or group of the stack is true as deleting from memory <> deleting from stack. I think this would probably avoid too many headaches from this change and avoid needing a force delete option. I'll need to look into how to do it but I suspect the cantDelete of a background shouldn't stop a card it is on from being deleted either.